test/test_torch.py

```diff
@@ -2248,6 +2248,13 @@ def test_eye(self):
         torch.eye(100, 100, out=res2)
         self.assertEqual(res1, res2)
 
+    def test_eye_like(self):
+        res1 = torch.eye(100, 100)
```

Comment:
Could you test the following cases please:
- torch.eye_like(long-rectangle-matrix)
- torch.eye_like(fat-rectangle-matrix)

Reply:
Isn't this backending to `eye`?

Reply:
True